src/mgr/BaseMgrStandbyModule.cc

```diff
@@ -65,7 +65,7 @@ ceph_get_module_option(BaseMgrStandbyModule *self, PyObject *args)
 {
   char *what = nullptr;
   char *prefix = nullptr;
-  if (!PyArg_ParseTuple(args, "ss:ceph_get_module_option", &what)) {
+  if (!PyArg_ParseTuple(args, "ss:ceph_get_module_option", &what, &prefix)) {
```

Comment:
Oh man, I complained about the Active-Standby mismatching interfaces, and overlooked the big thing here :man_facepalming: 

Nevertheless, I've filed a separate issue for refactoring this, as the amount of duplicate code in this area is high: http://tracker.ceph.com/issues/38717